recipes/gcc/gcc.inc

```diff
@@ -48,6 +48,7 @@ PROVIDES_${PN}[qa] += "allow-missing-provides:liblto-plugin"
 # Get rid of gcc-dev package
 PACKAGES = "${PN}-dbg ${PN}-doc ${PN}-locale ${PN}"
 FILES_${PN} += "${libdir}/*.a"
+FILES_${PN} += "${libdir}64/*.a"
```

Comment:
Please move this commit to separate commit (together with the similar binutils commit).

We need to investigate what is going on here, but let's avoid blocking the rest of this PR by that.